instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/engine.py

```diff
@@ -92,20 +147,27 @@ def _wrap_connect_internal(func, module, args, kwargs):
 
 
 class EngineTracer:
-    def __init__(
-        self, tracer, engine, enable_commenter=True, commenter_options=None
-    ):
+    def __init__(self, tracer, engine, options: EngineTracerOptions):
         self.tracer = tracer
         self.engine = engine
         self.vendor = _normalize_vendor(engine.name)
-        self.enable_commenter = enable_commenter
-        self.commenter_options = commenter_options if commenter_options else {}
-
+        self.enable_commenter = options.get("enable_commenter", False)
```

Comment:
I think at that point these keys should always be in the dictionnary, so I would rather use `options[...]`.